_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme

```diff
@@ -562,6 +572,17 @@ function rhdp2_preprocess_field(&$variables, $hook) {
       }
     }
   }
+
+  // Turn AddEvent ID into links
+  if ($variables['field_name'] === 'field_ics_addevent_id') {
+    $addevent_id = $variables['element']['#object']->get('field_ics_addevent_id')->getValue()[0]['value'];
```

Comment:
Can we validate this AddEvent ID? For example, does it always just contain a-z A-Z 0-9?

Reply:
@jordanpagewhite I'm not sure, I've only got the one example. I think we'd need to reach out to whoever is giving us those links to confirm. I could at least validate that we're not getting a URL passed or really anything with a forward slash.

Reply:
I confirmed that this can be limited to an alphanumeric value and have a commit adding validation.